tests/conftest.py

```diff
@@ -327,8 +363,13 @@ async def five_nodes(db_version: int, self_hostname, blockchain_constants):
 
 
 @pytest_asyncio.fixture(scope="function")
-async def wallet_nodes():
-    async_gen = setup_simulators_and_wallets(2, 1, {"MEMPOOL_BLOCK_BUFFER": 1, "MAX_BLOCK_COST_CLVM": 400000000})
+async def wallet_nodes(blockchain_constants):
+    constants = blockchain_constants
+    async_gen = setup_simulators_and_wallets(
+        2,
+        1,
+        {"MEMPOOL_BLOCK_BUFFER": 1, "MAX_BLOCK_COST_CLVM": 400000000, "SOFT_FORK3_HEIGHT": constants.SOFT_FORK3_HEIGHT},
```

Comment:
if you make this fixture parameterized by `blockchain_constants`, it doesn't look right to *only* forward the soft-fork3 height. there's no parameter to pass in the whole constants object?